Installation/installation.json

```diff
@@ -145,7 +145,17 @@
         "actionHandler": "CommonGateway\\XxllncZGWBundle\\ActionHandler\\ZGWToXxllncHandler",
```

Comment:
Note that this only works if you only need 1 action for this 1 ActionHandler, I always recommend using a separate action.json file instead